src/index.js

```diff
@@ -32,13 +32,18 @@ module.exports = async app => {
 		const added = context.payload.repositories_added;
 
 		added.forEach(async repository => {
-			// Save this new repository.
-			repositoryStore.set(repository.id, {
-				id: repository.id,
-				name: repository.name
-			});
+			if (repository.archived) {
```

Comment:
I don't believe this property exists on repositories for this event.

I'll add more detail as a comment to the test.